### PR TITLE
Migrate cex_ethereum.flows to a sector spell with macros to easily add new chains

### PR DESCRIPTION
**Pull request by hildobby**
Goal here is to create this macro so i can easily add cex_{chain}.flows for all other evm chains afterwards @jeff-dude

### CODE REVIEW

File: macros/models/_sector/cex/cex_flows.sql [hunk 1]
```diff
@@ -29,8 +21,10 @@ SELECT t.blockchain
 , t.tx_hash
 , t.evt_index
 , t.unique_key
-FROM {{ ref('tokens_ethereum_transfers')}} t
-INNER JOIN {{ ref('cex_ethereum_addresses')}} a ON a.address IN (t."from", t.to)
+FROM {{transfers}} t
+INNER JOIN {{addresses}} a ON a.address IN (t."from", t.to)
```
Comment (jeff-dude) [inline macros/models/_sector/cex/cex_flows.sql:25]: can you run a quick test on performance to see how runtime looks when you remove `IN` and do `(a.address = t."from" or a.address = t.to)`?

**Review by jeff-dude — COMMENTED** [review]
music to my ears!

one quick comment below, if you don't mind checking performance

File: macros/models/_sector/cex/cex_flows.sql (hunk 1 above)
Comment (jeff-dude) [reply macros/models/_sector/cex/cex_flows.sql]: old run for reference:
https://github.com/duneanalytics/spellbook/actions/runs/8178326082/job/22361994788

can compare when new is done

Comment (jeff-dude) [reply macros/models/_sector/cex/cex_flows.sql]: okay, no real change. feel free to keep as-is or revert to your preference

Comment (hildobby) [reply macros/models/_sector/cex/cex_flows.sql]: it's good as is, feel free to merge

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**